src/pages/Groups/Group1/Screen2.tsx

```diff
@@ -1,7 +1,140 @@
+  Timing_container,
+  Bookings_container,
+  IndividualBookings,
+  TopNav,
+  Header,
+  EventText,
+  EventTextWeekly,
+  TimeText,
+  LocationText,
+  LocationTextWeekly,
+  BotNav,
+  StyledButton,
+  ButtonContainer,
+  ToggleButton,
+  Calendar_container,
+  Day_container,
+  Days,
+  EventCard,
+} from './styles/Screen2.styled'
+import homeIconUnselected from '../../../assets/Group1/homeIconUnselected.svg'
+import facilitiesIconUnselected from '../../../assets/Group1/facilitiesIconUnselected.svg'
+import calenderIconSelected from '../../../assets/Group1/calenderIconSelected.svg'
+import profileIconUnselected from '../../../assets/Group1/profileIconUnselected.svg'
+import { useState } from 'react'
 
 const Screen2 = () => {
-  return <>this is screen 2</>
+  const [isDailyMode, setDailyMode] = useState(false)
+
+  return (
+    <>
+      {renderTopNav(isDailyMode)}
+
+      <Bookings_container>
+        {/*TODO: Create new component*/}
+        {/*{console.log(mockValues)}*/}
+        <ButtonContainer>
+          <ToggleButton onClick={() => setDailyMode(!isDailyMode)}>
+            {isDailyMode ? 'Show Weekly' : 'Show Daily'}
+          </ToggleButton>
+        </ButtonContainer>
+        {isDailyMode ? renderDailySchedule() : renderWeeklySchedule()}
+      </Bookings_container>
+      {renderBotNav()}
+    </>
+  )
 }
 
+const renderTopNav = (isDailyMode: boolean) => (
+  <TopNav>
+    <LeftOutlined />
+    <Header>{isDailyMode ? 'Oct 8, Friday (Semester 1, Week 7)' : 'Semester 1, Week 7'}</Header>
+    <UnorderedListOutlined />
+  </TopNav>
+)
+const renderBotNav = () => (
+  <BotNav>
+    <StyledButton src={homeIconUnselected} />
+    <StyledButton src={facilitiesIconUnselected} />
+    {/*TODO: Get designers to get a green selected svg for this icon*/}
+    <StyledButton src={calenderIconSelected} />
+    <StyledButton src={profileIconUnselected} />
+  </BotNav>
+)
+
+const renderDailySchedule = () =>
+  mockValues.map((e) => (
+    <IndividualBookings background={e.type} key={e.id}>
+      <EventText>{e.event}</EventText>
+      <TimeText>
+        {e.time.start} - {e.time.end}
+      </TimeText>
+      <LocationText>{e.location}</LocationText>
+    </IndividualBookings>
+  ))
+
+// TODO: Finish this part
+const renderWeeklySchedule = () => (
+  <WeekContainer>
+    <Timing_container>
+      {timings.map((t) => (
+        <Time_container key={t}>{t}</Time_container>
+      ))}
+    </Timing_container>
+    {days.map((d) => (
+      <>
+        <Calendar_container key={d.id}>
+          <Days>{d.day}</Days>
+          {timings.map((t) => (
+            <Day_container key={t.toString() + d.day} day={d.day}>
+              {d.day === 'Fri' ? mapEvents(t) : null}
+            </Day_container>
+          ))}
+        </Calendar_container>
+      </>
+    ))}
+  </WeekContainer>
+)
+
+const mapEvents = (t: string) => mockValues.map((e) => (e.time.start === t ? renderEventCard(e) : null))
+
+const renderEventCard = (e: any) => {
```

Comment:
"any" is ok here but fyi we usually define a type to avoid runtime errors.

For example:
`type Person = {
  name: string;
  age: number;
};`